### PR TITLE
Move template field projection ids into meeting_user

### PR DESCRIPTION
**Pull request by reiterl**
Update model, actions, tests

### CODE REVIEW

File: openslides_backend/action/actions/meeting/export_helper.py [hunk 1]
```diff
@@ -120,11 +120,13 @@ def export_meeting(datastore: DatastoreService, meeting_id: int) -> Dict[str, An
...
                     field_name = user_field.get_own_field_name()
-                    if (
-                        entry.get(field_name)
-                        and collection_from_fqid(entry[field_name]) == "user"
-                    ):
+                    if not entry.get(field_name):
+                        continue
+                    if collection_from_fqid(entry[field_name]) == "user":
                         user_ids.add(id_from_fqid(entry[field_name]))
+                    if collection_from_fqid(entry[field_name]) == "meeting_user":
```
Comment (r-peschke) [inline openslides_backend/action/actions/meeting/export_helper.py:127]: elif instead if: can't be both

**Review by r-peschke — CHANGES_REQUESTED** [review]
just a very small one;-)

**Review by r-peschke — CHANGES_REQUESTED** [review]

File: openslides_backend/presenter/get_user_related_models.py [hunk 2]
```diff
@@ -151,18 +151,19 @@ def get_meetings_data(self, user_id: int) -> List[Dict[str, Any]]:
...
             meeting_users = self.datastore.filter(
-                "meeting_user", filter_, ["speaker_ids", "submitted_motion_ids"]
+                "meeting_user",
+                filter_,
+                ["speaker_ids", "submitted_motion_ids", "assignment_candidate_ids"],
             )
             speaker_ids = []
             submitter_ids = []
+            candidate_ids = []
             if meeting_users:
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/presenter/get_user_related_models.py:162]: See 1538.

File: openslides_backend/presenter/get_user_related_models.py [hunk 3]
```diff
@@ -151,18 +151,19 @@ def get_meetings_data(self, user_id: int) -> List[Dict[str, Any]]:
...
                 for meeting_user in meeting_users.values():
                     speaker_ids = meeting_user.get("speaker_ids", [])
                     submitter_ids = meeting_user.get("submitted_motion_ids", [])
+                    candidate_ids = meeting_user.get("assignment_candidate_ids", [])
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/presenter/get_user_related_models.py:166]: See 1538

File: openslides_backend/action/actions/meeting/export_helper.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/meeting/export_helper.py]: Done.

File: openslides_backend/models/models.py [hunk 4]
```diff
@@ -3,7 +3,7 @@
 from openslides_backend.models import fields
 from openslides_backend.models.base import Model
 
-MODELS_YML_CHECKSUM = "fa09f258347319e4db20d5837bab2a5b"
```
Comment (r-peschke) [inline openslides_backend/models/models.py:6]: Must be generated anyway

**Review by r-peschke — APPROVED** [review]

File: openslides_backend/models/models.py (hunk 4 above)
Comment (reiterl) [reply openslides_backend/models/models.py]: Regenerated the models.py

**Pull request merged**